app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -251,11 +257,15 @@ class Core(
     }
 
     val adsTelemetry by lazy {
-        AdsTelemetry(context.components.analytics.metrics)
+        AdsTelemetry(metrics)
     }
 
     val searchTelemetry by lazy {
-        InContentTelemetry(context.components.analytics.metrics)
+        InContentTelemetry(metrics)
+    }
+
+    val metrics by lazy {
```

Comment:
Nit: move this above `adsTelemetry` and `searchTelemetry`

Reply:
OK, moved :).